index.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/4e655bdddb.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <!--header section starts-->
+    <header>
+        <h2 class="logo"><a href="#">SP</a></h2>
+        <nav>
+            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" clip-rule="evenodd"
+                    d="M3 7C3 6.44772 3.44772 6 4 6H20C20.5523 6 21 6.44772 21 7C21 7.55228 20.5523 8 20 8H4C3.44772 8 3 7.55228 3 7ZM3 12C3 11.4477 3.44772 11 4 11L20 11C20.5523 11 21 11.4477 21 12C21 12.5523 20.5523 13 20 13L4 13C3.44772 13 3 12.5523 3 12ZM4 16C3.44772 16 3 16.4477 3 17C3 17.5523 3.44772 18 4 18H20C20.5523 18 21 17.5523 21 17C21 16.4477 20.5523 16 20 16H4Z"
+                    fill="#6070FF" />
+            </svg>
+        </nav>     
+    </header>
+    <!--header section ends-->
+
+    <!--headline section starts-->
+    <section class="headline-main">
+        <section class="headline">
+            <h2 class="headline-heading">I'm Suthish Prasanna<br>Glad to see you!</h2>
+            <p class="headline-paragraph">I’m a software developer! I can help you build a product , feature or website
+                Look
+                through some of my work
+                and experience! If you like what you see and have a project you need coded, don’t hestiate to contact
+                me.
+            </p>
+            <p class="headline-sm-suport">LET’S CONNECT</p>
+            <ul class="social-icons">
+                <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li> 
+                <li><a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
+            </ul>
+        </section>
+    </section>
+ <!--headline section ends-->
+
+<!--Card works starts-->
+ <section class="works-card">
```

Comment:
As requested by the previous reviewers, it is required to have a display of grid on the containing element for the project cards and a display of flex on each project card.

You find this hard to implement because there's is no containing element for the project cards. Kindly enclose all the cards within a section tag and give it a display of grid.

Reply:
I updated as mentioned thanks for the suggesion.